css/compositing/parsing/background-blend-mode-computed.html

```diff
@@ -29,9 +29,9 @@
 test_computed_value("background-blend-mode", "color");
 test_computed_value("background-blend-mode", "luminosity");
 
-test_computed_value("background-blend-mode", "normal, luminosity");
-test_computed_value("background-blend-mode", "screen, overlay");
-test_computed_value("background-blend-mode", "color, saturation");
+test_computed_value("background-blend-mode", "normal, luminosity", "normal");
+test_computed_value("background-blend-mode", "screen, overlay", "screen");
+test_computed_value("background-blend-mode", "color, saturation", "color");
```

Comment:
Can we add a comment saying we drop excess values?

Reply:
done! Thanks for the review.